src/yetibot/core/observers/karma.clj

```diff
@@ -0,0 +1,78 @@
+(ns yetibot.core.observers.karma
+  (:require
+   [taoensso.timbre :as log]
+   [clojure.string :as str]
+   [yetibot.core.chat :refer [chat-data-structure]]
+   [yetibot.core.hooks :refer [obs-hook]]
+   [yetibot.core.commands.karma :as karma]))
+
+(defn- emoji-shortcode->reaction
+  [emoji-shortcode]
+  (-> (re-matches #":(.+):" emoji-shortcode)
+      second
+      (str/replace "_" " ")))
```

Comment:
I didn't realize we had to remove `_`s 🤔
So `:thinking_face:` becomes `thinkingface`? Slack react API requires this?

Reply:
No, not required; I did it to be consistent since we're now delivering the other emojis as short codes.  Do you think it should be reverted back to the Unicode char?